src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/UrnSpec.scala

```diff
@@ -51,7 +50,13 @@ class UrnSpec extends FlatSpec with Matchers {
   it should "be false when it has non-digit characters" in {
     Urn.isValid("((TRUST80000001") shouldBe false
   }
+  it should "be false when its uppercase letters" in {
```

Comment:
"be false when its all letters (the regex is case-insensitive so you can also pass in Urn("aatrust12345678" and this would be true)